ntoskrnl/include/internal/i386/ke.h

```diff
@@ -22,6 +22,30 @@ extern "C"
 #define KD_BREAKPOINT_SIZE        sizeof(UCHAR)
 #define KD_BREAKPOINT_VALUE       0xCC
 
+/* CPUID 1 - EDX flags */
+#define X86_FEATURE_FPU         0x00000001 /* x87 FPU is present */
```

Comment:
These defines are usually separated by the register (eax/ebx/ecx/edx) and the leaf. So this would be something like `X86_CPUID_STD_EDX_FPU`. And NX bit below would be `X86_CPUID_EXT_EDX_NX`

Reply:
Yes, this would be more clear, however I decided to follow how Timo did it for AMD64: https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/include/internal/amd64/ke.h;hb=40864bc15cb77ceb07e73f8625486c120192bb18#l29